app/lib/discourse_automation/scriptable.rb

```diff
@@ -125,10 +125,38 @@ def utils
     end
 
     module Utils
+      def self.fetch_report(name, args = {})
+        report = Report.find(name, args)
+
+        return nil if !report
```

Comment:
```suggestion
        return if !report
```
nil is implicitly returned.